Open Judge System/Web/OJS.Web/Controllers/BaseController.cs

```diff
@@ -88,6 +90,10 @@ protected override void OnException(ExceptionContext filterContext)
 
                 if (exception != null)
                 {
+                    this.Response.Clear();
+                    this.Response.TrySkipIisCustomErrors = true;
+                    this.Response.AddHeader("content-type", JsonMimeType);
+                    this.Response.Write(exception.Message);
```

Comment:
This is major change and should be tested very carefully across all the system where ajax requests are made (including Kendo requests). IMO it is not worth it for this feature. Just return "error" if no other solution.